nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -34,6 +34,24 @@
 from nncf.common.utils.timer import timer
 
 
+def dump_parameters(model: ov.Model, parameters: Dict, path: Optional[List] = None) -> None:
+    """
+    Dumps input parameters into Model's meta section.
+
+    :param model: ov.Model instance.
+    :param parameters: Incoming dictionary with parameters to save.
+    :param path: Optional list of the paths.
+    """
+    path = path if path else []
+    for key, value in parameters.items():
+        # Special condition for composed fields like IgnoredScope
+        if isinstance(value, IgnoredScope):
+            dump_parameters(model, value.__dict__, [key])
+            continue
+        rt_path = ['nncf', 'quantization'] + path + [key]
+        model.set_rt_info(str(value), rt_path)
```

Comment:
Please add try/catch block for these code.

Reply:
Why do we need to add try/catch here? Since the value is a string, the only thing that can be wrong is adding to the incorrect field.

Reply:
Added.